portfolio/src/main/webapp/style.css

```diff
@@ -103,6 +103,12 @@ li{
     width:50px;
 }
 
+#greeting {
+    color:#ffffff;
```

Comment:
fyi: Theres a short hand you can use here `#fff` if you'd like (I know plenty who prefer to use the full color). Basically if you use only 3 digits it doubles each one so for example `#3f8` would translate to `#33ff88`